apps/client/src/common/utils/socket.ts

```diff
@@ -14,7 +14,7 @@ import {
 } from '../stores/clientStore';
 import { addDialog } from '../stores/dialogStore';
 import { addLog } from '../stores/logger';
-import { patchRuntime, patchRuntimeProperty } from '../stores/runtime';
+import { addToBatchUpdates, flushBatchUpdates, patchRuntime, patchRuntimeProperty } from '../stores/runtime';
```

Comment:
Oh, I mean that the batch - flush could be on the server.

I assume that if the server sends all the messages when it is done, they will arrive within some milliseconds of each other

I wonder if this way we could have updates arriving out of order and not being flushed which would cause some weird state bugs?